apps/block_scout_web/assets/js/omni.js

```diff
@@ -0,0 +1,163 @@
+// local cache to re-use data
+let txMap = {};
+
+// default api url point to local server
+let url = "http://localhost:8888";
+
+const customMarker = () => {
```

Comment:
is this xchain marker just adding xchain image or something else? maybe lets name it like that

Reply:
as the name suggests it gives the custom marker for xchain on txs, it's more than an image.

Reply:
will rename to xchainMarker()